Src/library/wel/windows/wel_window.e

```diff
@@ -2774,6 +2786,24 @@ feature {WEL_INPUT_EVENT} -- Externals
 			"InvalidateRect"
 		end
 
+feature -- Externals DPI
+
+	cwin_get_dpi_for_window (hwnd: POINTER): INTEGER
+		external
+			"C [macro %"wel.h%"] (HWND): EIF_INTEGER"
+		alias
+			"GetDpiForWindow"
+		end
+
+	cwin_mul_div (a_init_value, a_dpi, a_val: INTEGER): INTEGER
```

Comment:
I was wondering, could we just use inline external ?
I feel the "macro" is the old way to do (I may be wrong).

Reply:
yes, I just follow the style of the WEL_WINDOW class.
I can update the code if needed.